.changeset/olive-comics-knock.md

```diff
@@ -0,0 +1,6 @@
+---
+"@comet/cms-api": minor
+"comet-docs": minor
+---
+
+Add AccessLog Interceptor
```

Comment:
Please be a little more specific: What is it, why do we add it, how can one integrate it into an application